src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useRef, ReactNode } from 'react';
+import { createPortal } from 'react-dom';
+import styles from '../CardsList/CardsList.module.css';
+
+const modalRoot = document.createElement('div');
+modalRoot.setAttribute('id', 'modal-root');
+document.body.appendChild(modalRoot);
```

Comment:
This -15 points.

Reply:
fixed with document.body this manipulation is allowed
![image](https://user-images.githubusercontent.com/77505619/231173189-2391daef-0b46-450a-852c-8ad12e84664d.png)